.github/workflows/backend.yml

```
@@ -0,0 +1,53 @@
+      
+    - name: Setup Rust
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        
+    - name: Check Rust code
+      run: |
+        cd upload-proxy
+        cargo check
+        echo "✅ Rust code compiles"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)